srml/offences/src/mock.rs

```diff
@@ -109,33 +113,43 @@ pub type System = system::Module<Runtime>;
 
 pub const KIND: [u8; 16] = *b"test_report_1234";
 
+/// Returns all offence details for the specific `kind` happened at the specific time slot.
+pub fn offence_reports(kind: Kind, time_slot: u128) -> Vec<OffenceDetails<u64, u64>> {
+	<crate::ConcurrentReportsIndex<Runtime>>::get(&kind, &time_slot.encode())
+		.into_iter()
+		.map(|report_id| <crate::Reports<Runtime>>::get(&report_id).unwrap())
```

Comment:
it's test code, but I think still `expect` could provide a better error message.